runtime/src/execute_cost_table.rs

```diff
@@ -106,6 +106,17 @@ impl ExecuteCostTable {
             }
         }
 
+        // Logs events of unusual input `value` or `aggregated.ema`
+        const MAX_COST: u64 = 200_000;
+        if let Some(aggregated) = self.table.get(key) {
```

Comment:
seems like this can live in the `Entry::Occupied` match arm above to avoid the second lookup

Reply:
moved